docs/_guide/styles.md

```diff
@@ -30,7 +30,7 @@ Shadow DOM scopes CSS so that styles defined in a shadow root only apply to DOM
 **If you're using the Shady CSS polyfill, be aware that it has some limitations.** See the [Shady CSS README](https://github.com/webcomponents/shadycss/blob/master/README.md#limitations) for more information.
 {.alert-info}
 
-## Developing a component
+## Developing a LitElement component
```

Comment:
"Developing" -> "Styling"?

Reply:
i want to specifically call out "developing" vs "consuming" - when you're developing a component you want to know how to mess around with shadow DOM styles etc, but when you're consuming, this should all be  a black box and you should just be styling my-element {...} from wherever you import it

Reply:
Yep I see that. Will think about it & update accordingly - thanks both!